docs/content/topics/operators/security.md

```diff
@@ -19,28 +19,35 @@ For that reason, it is prudent to configure security.
   in a Helm install, do `helm install --namespace brigade ...`.
 - *RBAC Enabled*: When installing with Helm, role-based access control is enabled by default.
   To disable,`--set rbac.enabled=false` will turn off role-based access control.
-- *Do not run more than one brigade per namespace*: Running multiple installs of Brigade
+- *Do not run more than one Brigade per namespace*: Running multiple installs of Brigade
   in the same namespace can cause naming collisions which could result in
   unauthorized access to pods.
-- *Do not run multiple tenants against the same brigade*: Brigade does not implement
-  security controls that allow multiple tenants to share the same Brigade instance.
-  Brigade supports multiple projects per Brigade server instance, but those projects
-  should be owned by the same tenant.
```

Comment:
I might be slightly hesitant to remove this outright, although we could perhaps reword it.

While we have much better access controls now, there could still be reasons to run a Brigade per team, for instance.

I say this because the `READER` role confers _global_ read access (with the exception of logs). This was a deliberate choice to avoid the very difficult problem of filtering project list results on the basis of which projects the user has access to... so a comparison could be made to GitHub. Every project could be likened to a _public_ repo. Everyone can see. Not everyone can write...

If for whatever reason that's unacceptable, then the practical solution is to run multiple Brigades.